src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<blockquote><a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/cf8fa880a6d979a72ead2cc6fe2c5613b7bd453d/src/fastoad/notebooks/02_explanation/modules/pure_python/geometry" target="_blank">The geometry folder</a> contains: </blockquote><p>Again, the link (to a folder) does not work. Maybe a link to the main script of the folder would do ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Other note to self, in the part on the MDO I say "More precise results will be obtained in the following notebooks", only the results are not especially more precise, only obtained with more "correct" methods.

Reply:
> Again, the link (to a folder) does not work. Maybe a link to the main script of the folder would do ?

Corrected in 3ba48f01

Reply:
<p>There are still hyperlinks for "geometry folder", and so on, that appear as non-working links.</p>

Reply:
Are you refering to the tooltips ? I cannot find any hyperlinks to folders anymore.
In any case I deleted the tooltips in 28459df2 so they are not mistaken for links

Reply:
<p>I mean this:</p><blockquote><a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/ebacceb09dcca546a65d45928c135377d8654e78/src/fastoad/notebooks/02_explanation/a" target="_blank">The geometry folder</a> contains: </blockquote><p>So, yes, these are the tooltips you proposed, and I had forgot it, and with my fresh new eye, I considered them as broken hyperlinks. </p><p>To me, that confirms what as I stated above: I do not think it is a proper solution.</p>

Reply:
Are you OK if I delete them ?

Reply:
<p>Yes. That was my suggestion in my last comment for the cell above (where the first tooltip is used), with maybe the folder location in plain text (<code>modules/pure_python</code>.)</p><p> </p><p> </p>

Reply:
<p>Added in 7a6643b4 what you suggested in a previous conversation, </p><p> </p>